wordpress-mobile/WordPress-iOS
# Implement Stock Photos from Media Library + button

Implements part of  #8954

In this PR: 
- Add support for Stock Photos to the Media Library, according to the following flow:

![37942004-9ce5361a-31a3-11e8-96ed-3954657f13a1](https://user-images.githubusercontent.com/2722505/38291341-1a438be8-3794-11e8-9771-7f9969bd5830.jpg)

I continued piggybacking on the work we had already done on the media picker. I renamed a bunch of the abstractions that we already had created while developing this feature. Turns out some naming was a bit too specific to Aztec. I am terrible at naming things, good thing there are code reviews.

To test:
1. Select a blog > Media
2. Tap the `+` button
3. Taking a Photo or Video should work
4. Importing from the Photo Library should work
5. Importing from Stock Photos should work (although we have not completely finished the implementation of that side of the UI)
6. Importing from Other apps should also work.
